_config.yml

```diff
@@ -20,6 +20,9 @@ collections:
   consulting:
     output: true
     permalink: /consulting/:path/
+  events:
+    output: true
```

Comment:
I am not sure we need this line

Reply:
Oh, the idea was to make Jekyll process each file within the `_events` collection, and display them an individual page(s).

Reply:
It works for posts and it does not have this line